vector/src/main/res/layout/keybackup_setup_step3_fragment.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/keybackupsetup_step3_root"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    tools:context=".fragments.keybackupsetup.KeybackupSetupStep3Fragment">
+
+    <TextView
+        android:id="@+id/keybackupsetup_step3_text"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:layout_marginTop="30dp"
+        android:layout_marginEnd="20dp"
+        android:text="@string/keybackup_setup_step3_text"
+        android:textAlignment="center"
+        app:layout_constraintBottom_toTopOf="@id/keybackup_recovery_key_container"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_chainStyle="spread_inside" />
+
+
+    <android.support.constraint.ConstraintLayout
+        android:id="@+id/keybackup_recovery_key_container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="20dp"
+        android:layout_marginBottom="20dp"
+        app:layout_constraintBottom_toTopOf="@id/keybackupsetup_step3_copy_button"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/keybackupsetup_step3_text">
+
+        <LinearLayout
```

Comment:
To remove, use ConstraintLayout capabilities, or remove the ConstraintLayout

Reply:
done (use power)